src/plugins/sailfishoswizards/SailfishOSWizards.json.in

```diff
@@ -0,0 +1,18 @@
+{
+    \"Name\" : \"SailfishOSWizards\",
```

Comment:
The name of the plugin should probably be "Sailfish Wizards", as it is not part of the OS. Just like we call the SDK "Sailfish SDK", not "Sailfish OS SDK"

Reply:
The same rule in code: replace namespace SailfishOSWizards to namespace SailfishWizards and other same?

Reply:
Yes. Only when you are referring to the OS, you should use "Sailfish OS". e.g. on line 15 "A set of usefull wizards that ease the development of the applications for Sailfish OS" is correct.

Reply:
Renamed